recipe/meta.yaml

```
@@ -1,41 +1,40 @@
-{% set version = "4.0.2" %}
+{% set version = "4.3.0" %}
 
 package:
   name: swig
   version: {{ version }}
 
 source:
-  fn: swig-{{ version }}.tar.gz  # [unix]
-    
-  url: http://prdownloads.sourceforge.net/swig/swig-{{ version }}.tar.gz  # [unix]
-  sha256: d53be9730d8d58a16bf0cbd1f8ac0c0c3e1090573168bfa151b01eb47fa906fc  # [unix]
-  fn: swigwin-{{ version }}.zip  # [win]
-  url: http://prdownloads.sourceforge.net/swig/swigwin-{{ version }}.zip  # [win]
-  sha256: daadb32f19fe818cb9b0015243233fc81584844c11a48436385e87c050346559  # [win]
+  url: https://github.com/swig/swig/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: f2136da1137a20dfcec795fe0d17ca1a2465d28e3b307f122526629b6b2f2294
 
 build:
-  number: 4
+  number: 0
   detect_binary_files_with_prefix: True  # [unix]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - bison # [unix]
-    - make  # [unix]
-    - libtool  # [unix]
+    - gnuconfig    # [unix]
+    - bison        # [unix]
+    - make         # [unix]
+    - automake     # [unix]
+    - pkg-config   # [unix]
+    - cmake        # [win]
+    - msys2-bison  # [win]
   host:
-    - pcre  # [unix]
-  run:      # [unix]
-    - pcre  # [unix]
+    - pcre2 10
+    - zlib 1.2
```

```suggestion
    - zlib {{ zlib }}
```
because it's in cbc.yaml https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L241